workloads/virtual-machines/create-networkpolicy.md

```diff
@@ -1,10 +1,10 @@
 # Enabling NetworkPolicy for VirtualMachine
 
-Before creating NetworkPolicy object, make sure you are using a networking solution which supports NetworkPolicy. Network isolation is controlled entirely by NetworkPolicy object. By default, all vms in a namespace are accessible from other vms and network endpoints. To isolate one or more vms in a project, you can create NetworkPolicy objects in that namespace to indicate the allowed incoming connections.
+Before creating NetworkPolicy object, make sure you are using a networking solution which supports NetworkPolicy. Network isolation is controlled entirely by NetworkPolicy object. By default, all vms/pods in a namespace are accessible from other vms/pods and network endpoints. To isolate one or more vms/pods in a project, you can create NetworkPolicy objects in that namespace to indicate the allowed incoming connections.
```

Comment:
Hm, instead of writing everywhere `vms/pods` I just had in mind that we clearly state with one or two explicit sentences that vms and pods are equivalent regarding to NetworkPolicies and services (Maybe even in fat).